db/fixtures/007_web_hook_event_types.rb

```diff
@@ -230,13 +230,3 @@
   b.name = "chat_message_restored"
   b.group = WebHookEventType.groups[:chat]
 end
-WebHookEventType.seed do |b|
-  b.id = WebHookEventType::TYPES[:category_experts_approved]
-  b.name = "category_experts_approved"
-  b.group = WebHookEventType.groups[:post]
-end
-WebHookEventType.seed do |b|
-  b.id = WebHookEventType::TYPES[:category_experts_unapproved]
-  b.name = "category_experts_unapproved"
-  b.group = WebHookEventType.groups[:post]
-end
```

Comment:
Do we need a DB migration that deletes those seeded types from the `web_hook_event_types` table?

Reply:
Thank you so much for your comment! Yup, I just added the migration. Sorry for not switching this PR to draft beforehand. I will add description on this now.

Reply:
Please let me know if the description needs more details. Thank you!

Reply:
I truly hope that my message does not bother you in any way, shape, or form...! It’s just that I find these topics really fascinating, and I would love to discuss them with someone who shares similar interests! Are you familiar with the exciting and sometimes bewildering world of cryptocurrencies, NFTs, and All the various Altcoins out there? It’s a rapidly evolving landscape, and every day there seems to be something new and groundbreaking happening.